### PR TITLE
Config updates to be reviewed

### PR DESCRIPTION
**Pull request by infojunkie**
@caiosba plz review updated example files (including comments)

### CODE REVIEW

File: config/config.yml.example [hunk 1]
```diff
@@ -1,107 +1,211 @@
...
-  pusher_cluster:
-  video_file_max_size:
-  uploaded_file_max_size:
-  image_min_width:
-  image_max_width:
-  image_min_height:
-  image_max_height:
+  elasticsearch_log: true
+  elasticsearch_sync: false
+  elasticsearch_cloud_id: 'qa:ZXUtd2VzdC0xLmF3cy5mb3VuZC5pbyQxZDkxYjk4YzZmZjQ0YjA0OGZhNDc2NzI4ZTQ2ZTAzMSRhMjE1ODRmZGVkOTM0MzFiYjkyOWU5OGQ3OWYyMzUxMQ=='
```
Comment (caiosba) [inline config/config.yml.example]: I don't see this `elasticsearch_cloud_id` configuration being used anywhere... what's it used for?

File: config/config.yml.example [hunk 2]
```diff
@@ -1,107 +1,211 @@
...
+  elasticsearch_user: 'elastic'
+  elasticsearch_password: 'HzzENVbibvxsl4LaykQqqOlW'
```
Comment (caiosba) [inline config/config.yml.example]: This is the QA password, I don't think we need a password for the local instance, but please confirm with Aaron

File: config/config.yml.example [hunk 3]
```diff
@@ -1,107 +1,211 @@
...
+  google_credentials_path: '/app/config/credentials.json'
+
+  # Smooch (Zendesk Sunshine Conversations) messaging platform gateway
+  #
+  # REQUIRED for Check Message
+  #
+  smooch_twitter_consumer_key: '<SMOOCH CONSUMER KEY>'
+  smooch_twitter_consumer_secret: '<SMOOCH CONSUMER SECRET>'
+  smooch_twitter_tier: 'sandbox'
+  smooch_twitter_env_name: 'checkmessage'
```
Comment (caiosba) [inline config/config.yml.example]: Just to clarify more, those are the settings of a Twitter app used for Check Message, for the Twitter DMs integration

File: config/config.yml.example (hunk 1 above)
Comment (infojunkie) [reply config/config.yml.example]: @huslage any idea ^ ?

File: config/config.yml.example (hunk 3 above)
Comment (infojunkie) [reply config/config.yml.example]: are `sandbox` and `checkmessage` hardcoded values or they need to be set on the Twitter or Smooch side as well ?

Comment (caiosba) [reply config/config.yml.example]: on Twitter side as well

**Pull request merged**